build/MacOSXBundleInfo.plist.in

```diff
@@ -447,7 +447,7 @@
 	<key>LSRequiresCarbon</key>
 	<true/>
 	<key>LSMinimumSystemVersion</key>
-	<string>10.7.0</string>
+	<string>10.10.0</string>
```

Comment:
Interesting, this is true ever since MuseScore 3.
And an even higher version will be needed for MuseScore 4.

Reply:
Yes, it has been there for very long. I just noticed it some time ago.
For MuseScore 4 I already updated it in PR #7169, to 10.14, after that Igor updated the deployment target in PR #7165 . It's quite a big leap, but future proof because it also supports Qt 6. Igor thought it was better to update immediately to 10.14, because if we would do 10.13 for MuseScore 4.0 (Qt 5.15) and then 10.14 for MuseScore 4.1 (Qt 6) that would be quite confusing. That seems indeed a wise decision to me.